src/QueryRunner.php

```diff
@@ -65,7 +65,7 @@ private function processScript(Block $block, Code $code, Script $script): void
         }
 
         // Skip select
-        if (strtoupper(substr($sql, 0, 6)) === 'SELECT') {
+        if (strtoupper(substr($sql, 0, 6)) === 'SELECT' && !strpos(strtoupper($sql), 'INTO')) {
```

Comment:
strpos treba kontrolovat ako `!== false` ... lebo vracia `0` ak je string na zaciatku, ... v tomto konkretnom pripade to nevadi, ....